virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,62 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.utils_test import libvirt
+from virttest.libvirt_xml import secret_xml
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(sec_dict=None, remote_args=None):
```

Comment:
To keep compatibility and agility, maybe below changes can be referred:
```
sec_default_dict = {'secret_ephemeral': 'no',
                    'secret_private': 'yes',
                    'description': 'secret desc',
                    'usage': 'volume',
                    'volume': '/path/to/volume'}

sec_default_dict.update(sec_dict)
secret_xml.set_attrs(sec_default_dict)
```

This can bring at least 3 positive effects:
1)sec_dict can be passed partially, not monolithic one
2)Even sec_dict is {}, it can use default value to create secret
3)I assume this help keep compatibility with old libvirt.create_secret() method

Reply:
@chunfuwen Thanks for your comment . As we discussed with dzheng ,yicui. I will remove default value and give example in doc string